### PR TITLE
unix, win: Added uv_has_ref function

### PR DESCRIPTION
**Pull request by saghul**
I deleted the other branch by mistake :-S

### CODE REVIEW

File: include/uv.h [hunk 1]
```diff
@@ -281,6 +281,7 @@
  */
 UV_EXTERN void uv_ref(uv_handle_t*);
 UV_EXTERN void uv_unref(uv_handle_t*);
+UV_EXTERN int uv_has_ref(uv_handle_t*);
```
Comment (bnoordhuis) [inline include/uv.h]: `const uv_handle_t*`?

File: test/test-ref.c [hunk 2]
```diff
@@ -413,3 +413,15 @@ static void prepare_cb(uv_prepare_t* h, int status) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+
+TEST_IMPL(has_ref) {
+  uv_idle_t h;
+  uv_idle_init(uv_default_loop(), &h);
+  uv_ref((uv_handle_t*)&h);
+  ASSERT(uv_has_ref((uv_handle_t*)&h)==1);
```
Comment (bnoordhuis) [inline test/test-ref.c]: Style: spaces around ==

**Comment by saghul** [comment]
@bnoordhuis force pushed addressing both comments.

**Comment by bnoordhuis** [comment]
Thanks Saúl, landed in a92b66f.

**Pull request closed without merging**